### PR TITLE
Bug 1587555: Switch ci to the community cluster

### PR DESCRIPTION
**Pull request by walac**

### CODE REVIEW

**Comment by djmitche** [comment]
I just enabled the community-tc-integration bot for this repo (and disabled the old one).

**Comment by djmitche** [comment]
```
{
  "AllOf": [
    "docker-worker:cache:docker-worker-garbage-*",
    "docker-worker:capability:privileged",
    "docker-worker:capability:device:loopbackAudio",
    "docker-worker:capability:device:loopbackVideo",
    "secrets:get:repo:github.com/taskcluster/docker-worker:ci-creds",
    "secrets:get:repo:github.com/taskcluster/docker-worker:pulse-creds"
  ]
}
```

We're naming secrets within the project, rather than the repo, now.  Something like `project/taskcluster/testing/docker-worker/{ci,pulse}-creds`?

I think the other scopes will need to be added in community-tc-config.

**Comment by walac** [comment]
> I think the other scopes will need to be added in community-tc-config.

I failed to find where I should add the scopes to `static/taskcluster/github`

**Comment by djmitche** [comment]
I think you'll want to add an additional stanza like
https://github.com/mozilla/community-tc-config/blob/185e6799de76997dd1d2a766230a06a7e28c6ad0/generate/repo_access.py#L26-L36
```
granting the necessary scopes to the docker-worker repo.

**Comment by djmitche** [comment]
oh I see https://github.com/mozilla/community-tc-config/pull/10 :)

**Comment by walac** [comment]
I disabled `dind` tests, for now, to move forward.

File: .taskcluster.yml [hunk 1]
```diff
@@ -30,7 +30,7 @@ tasks:
         taskclusterProxy:
           true
       env:
-        TASKCLUSTER_ROOT_URL: https://taskcluster.net
```
Comment (djmitche) [inline .taskcluster.yml]: The worker *should* set this for you anyway?  But, doesn't hurt..

File: .taskcluster.yml [hunk 2]
```diff
@@ -43,24 +43,24 @@ tasks:
           export DOCS_PROJECT=docker-worker DOCS_TIER=workers DOCS_FOLDER=docs DOCS_README=README.md &&
           upload-project-docs
     metadata:
-      name: "Publish docker-worker docs to https://docs.taskcluster.net/reference/workers/docker-worker"
-      description: "Upload docker-worker documentation to taskcluster [docs site](https://docs.taskcluster.net/reference/workers/docker-worker)."
+      name: "Publish docker-worker docs to https://docs.community-tc.services.mozilla.com/reference/workers/docker-worker"
+      description: "Upload docker-worker documentation to taskcluster [docs site](https://docs.community-tc.services.mozilla.com/reference/workers/docker-worker)."
       owner: '{{ event.head.user.email }}'
       source: '{{ event.head.repo.url }}'
```
Comment (djmitche) [inline .taskcluster.yml]: This whole task actually doesn't do anything anymore, and could be removed (but again doesn't hurt)

**Pull request merged**